src/Pages/Deals.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import Catalog from "../Components/Catalog/Catalog"
+import Api from "../Lib/Api"
+
+export default class Items extends Component {
+  constructor() {
+    super()
+    this.state = {
+      items: []
+    }
+  }
+  render() {
+    return (
+      <div>
+        <Catalog items={this.state.items} message="Currenlty no deals" />
+      </div>
+    )
+  }
+  async componentDidMount() {
+    let items = await Api.getDeals();
+    this.setState({ items: items })

```

Comment:
you dont have a semi-colon here which probably means you dont have prettier setup